### DISCUSSION TITLE
Help needed, deleted cockpit in proxmox by error..

### DISCUSSION THREAD

**Discussion by maddog87** [cockpit-project/cockpit#21302] [Q&A]
Hi 
I am a noob when it comes to this so please help.
I was having a error with cockpit so deleted the container in proxmox, I have reinstalled it in the same container and set up as it was before.
The issue is I can't find any of the data that was in it. And my other containers like Jellyfin cant find the data it use to see.
Any suggestions on how to get the data on the storage disc to show up again?
I didnt have a backup.

**Comment by jelly** [comment]
I am not sure if I can follow your issue, you deleted a container which ran containers in it or am I understanding you incorrectly?

As we don't know how you made a container it is hard to tell if the data is still there

**Reply by maddog87** [reply]
So I deleted the Cockpit container, when I reinstalled Cockpit all my apps (Jellyfin, Sonaar, Radaar etc) went back to defaults. And Jellyfin can't see the folders that contained my media. When I try and point it to where it is there is nothing coming up. In cockpit navigator the folders that I use to have dont exist anymore. Not sure if the data is gone for good or just somehow hidden
